### ISSUE TITLE
A11y_AzureDataStudioStayHealthy_Welcome_AzureDataStudio-Insiders_Keyboard:Tab focus on non text elements.

### ISSUE THREAD

**Issue by fsteffi** [microsoft/azuredatastudio#10615]
[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:**#A11y_AzureDataStudioStayHealthy;#A11ySev2;#A11yTCS;#A11yMAS;#Benchmark#DesktopApp;#MAS2.1.1;#SQL Azure Data Studio;#FTP;#STP;#Keyboard;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.19.0-insider
Commit: e8dc0d15b78ef0e3f4ea127bb9eb7a6de492b221
Date: 2020-05-27T17:24:57.733Z
VS Code: 1.46.0
Electron: 7.2.4
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.18363

**Additional Details:**
MAS References:MAS2.1.1

**Repro Steps:**
1 .Launch Azure Data Studio Insiders application.
2 . If Welcome screen is not shown on application launch, then navigate to Help --> Welcome.
3 .Tab to Preview button and press enter.

**Actual:**
When the dialog opens twice tab key focus move to non-control elements.

**Expected:**
The text present in the dialog should not receive tab focus when the dialog opens the tab focus can be on the close button. Screen reader keys(insert+nupad for NVDA) and (alt+arrow key for JAWS) should announce the text data.

**User Impact:**
Keyboard and screen reader users will think the data is either editable or actionable.

**Attachment link for Reference:**
[#10615A11y_AzureDataStudioStayHealthy_Welcome_AzureDataStudio-Insiders_Keyboard.zip](https://github.com/microsoft/azuredatastudio/files/4700486/10615A11y_AzureDataStudioStayHealthy_Welcome_AzureDataStudio-Insiders_Keyboard.zip)

**Comment by alanrenmsft** [comment]
I think we don't need to spend time out fixing this issue, because the Preview button will be removed once the accessibility testing is done. so closing the issue.

**Comment by fsteffi** [comment]
![image](https://user-images.githubusercontent.com/24580570/223413279-c3690aac-235b-4fc6-975a-5fa2f31af5d2.png)

#Closed;
